X16 Reference - 09 - VERA Programmer's Reference.md

```diff
@@ -455,11 +462,28 @@ The VERA consists of:
 | $1FA00 - $1FBFF | Palette                    |
 | $1FC00 - $1FFFF | Sprite attributes          |
 
+The X16 KERNAL uses the following video memory layout:
+
+| Addresses     | Description                                               |
+|---------------|-----------------------------------------------------------|
+| $00000-$12BFF | 320x240@256c Bitmap                                       |
+| $12C00-$12FFF | *unused* (1024 bytes)                                     |
+| $13000-$1AFFF | Sprite Image Data (up to $1000 per sprite at 64x64 8-bit) |
+| $1B000-$1EBFF | Text Mode                                                 |
+| $1EC00-$1EFFF | *unused* (1024 bytes)                                     |
+| $1F000-$1F7FF | Charset                                                   |
+| $1F800-$1F9BF | *unused* (1024 bytes)                                     |
+| $1F9C0-$1F9FF | VERA PSG Registers (16 x 4 bytes)                         |
+| $1FA00-$1FBFF | VERA Color Palette (256 x 2 bytes)                        |
+| $1FC00-$1FFFF | VERA Sprite Attributes (128 x 8 bytes)                    |
+
+Application software is free to use any part of video RAM if it does not use the
+corresponding KERNAL functionality. To restore text mode, call `CINT` ($FF81).
+
 ***Important note:
 Video RAM locations 1F9C0-1FFFF contain registers for the PSG/Palette/Sprite attributes. Reading anywhere in VRAM will always read back the 128kB VRAM itself (not the contents of the (write-only) PSG/Palette/Sprite attribute registers). Writing to a location in the register area will write to the registers in addition to writing the value also to VRAM. Since the VRAM contains random values at startup the values read back in the register area will not correspond to the actual values in the write-only registers until they are written to once.
 Because of this it is highly recommended to initialize the area from 1F9C0-1FFFF at startup.***
```

Comment:
This is a relic of VERA docs being separate from X16 docs.  The X16 takes care of this initialization.